docs/product-assembly_dm-on-openshift/src/main/asciidoc/preparing-dm-openshift-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='preparing-dm-openshift-con']
+= Preparing to deploy {PRODUCT} in your OpenShift environment
+
+Before deploying {PRODUCT} in your OpenShift environment, you need to complete several preparatory tasks. You do not need to redo these tasks if you want to deploy additional images, for example, for new versions of decision services or for other decision services. 
+
+== Before you begin
```

Comment:
Again, template thing. Change all 4 instances of "== Before you begin" in this doc to ".Prerequisites", use a bulleted list for multiple items, and use either imperative statements or passive requirements.

Here for example, should  either this (active):

.Prerequisites
- Ensure that at least 4 gigabytes of memory are available in the OpenShift environment.
- Create the OpenShift project for the deployment. 
- Log in to the project using the web console or the `oc` command, as appropriate for the task.

Or this (passive):

.Prerequisites
- At least 4 gigabytes of memory are available in the OpenShift environment.
- The OpenShift project for the deployment has been created. 
- You are logged in to the project in the web console or the `oc` command, as appropriate for the task.

To date, CCS/BXMS style for prereqs is to use the passive (second example) rather than the active. Emily and I have been discussing this and she'll be standardizing in our team SG soon. There are arguments for both sides and we actually prefer the active, but the main value of the passive is that it works well when things need to have been done and not necessarily by the user, and leans toward them already having been done. Across the board, it seems more fitting in general.

So the choice is yours if you feel the active is better in certain cases, but it seems in general we're rolling with the passive approach, just fyi. Feel free to reach out to Emily with additional/differing thoughts.